.github/ISSUE_TEMPLATE/suggest-a-feature.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Suggest a Feature
+about: Suggest a new feature for Harper to improve its functionality
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+## Note
+
+This template is for suggesting new features for Harper. If you want to report a grammatical error that Harper doesn't detect, please use the [Report Grammatical Error](https://github.com/Automattic/harper/issues/new?template=report-grammatical-error.md) 
+template instead.
+
+## Description
+
+Provide a detailed description of the feature you would like to see in Harper. Explain what problem it would solve or what benefit it would bring to users.
+
+## Component
+
+Which component of Harper would this feature affect? For example:
+- Core grammar checking engine
+- Obsidian plugin
+- VS Code extension
+- Neovim plugin
+- WebAssembly version
```

Comment:
I think putting `harper.js` here would be better than WebAssembly version since that's the main way to consume Harper in JS land and AFAIK `harper-wasm` isn't officially published.